modules/install-osp-dualstack.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: CONCEPT
+[id="install-osp-dualstack_{context}]
+= Optional: Configuring a cluster with dual-stack networking
+
+:FeatureName: Dual-stack configuration for OpenStack
+include::snippets/technology-preview.adoc[]
+
+You can create a dual-stack cluster on {rh-openstack}. However, this is enabled only if you are using an {rh-openstack} network with IPv4 and IPv6 subnets. Before creating a dual-stack cluster, you must complete the following tasks:
+
+* Creating API and Ingress VIPs ports, and specifying the addresses in the `install-config.yaml`.
+* Adding the IPv6 subnet to a neutron router to enable router advertisements.
+* Ensuring the dualstack network MTU accommodates both the minimum MTU for IPv6, which is 1280, and the OVN-Kubernetes encapsulation overhead, which is 100.
```

Comment:
also, providing dual-stack network in the install-config

Reply:
Isn't this a part of deployment process?
This section lists 'prerequisites' for deployment, which means, things that has to be in place before we configure dual stack network.
Since the whole topic talks about only configuring dual stack network, the final step is updating install-config. Hence creating network and subnets become prereq along with MTU and other considerations.
Let me know your thoughts here.

Reply:
To achieve dual-stack OCP cluster you must have the following available before specifying the resources IDs on install-config and triggering installation:

- network created with ipv4 and ipv6 subnets and appropriate mtu
- subnet v6 added to the router
- api and ingress ports created

Those requirements are specific to dual-stack.

Reply:
@MaysaMacedo can you give this another look? I have modified as per your suggestions